ui/theme/theme.go

```diff
@@ -160,6 +170,26 @@ func (m *MyTheme) ListThemeFiles() map[string]string {
 	return result
 }
 
+func (m *MyTheme) createThemeResources() error {
+	err := configdir.MakePath(m.themeFileDir)
```

Comment:
you can use `os.MkdirAll` here and remove the dependency on configdir from this file